package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
The dev script doesn't seem to be working. Its starting the sdk but not the playground

Reply:
![image](https://github.com/user-attachments/assets/bbb2351a-fb5f-4248-bf99-948f21ae1f9c)
fixed